### PR TITLE
Add support for pfctl ruleset with anchors

### PR DESCRIPTION
**Pull request by TuxPowered42**

### CODE REVIEW

File: src/lbpool_states.py [hunk 1]
```diff
@@ -131,130 +118,92 @@ def main():
...
-def pfctl_parser(pfctl_output):
+def get_state_limit():
+    # The default limit set in pf.conf, fetch it from what is loaded in pf
+    default_limits = subprocess.run(
+        ['sudo', 'pfctl', '-sm'],
+        capture_output=True, text=True,
+    ).stdout.splitlines()
+
+    for line in default_limits:
+        if "states" in line:
```
Comment (lamaral) [inline src/lbpool_states.py:180]:
```suggestion
        if 'states' in line:
```

File: src/lbpool_states.py [hunk 2]
```diff
@@ -131,130 +118,92 @@ def main():
...
-
+    pfctl_output = subprocess.run(
+        ['sudo', 'pfctl', '-vsr', '-a*'],
+        capture_output=True, text=True,
+    ).stdout.splitlines()
+
+    ret = defaultdict(int)
+    for line in pfctl_output:
+        indx = pfctl_output.index(line)
+        if "route-to" not in line:
```
[the remark this answers is not in the document]
Comment (TuxPowered42) [reply src/lbpool_states.py:199]: Fixed.

File: src/lbpool_states.py (hunk 1 above)
Comment (TuxPowered42) [reply src/lbpool_states.py:180]: Fixed.

File: src/lbpool_states.py [hunk 3]
```diff
@@ -131,130 +118,92 @@ def main():
...
+        if exit_code == 0:
+            status = f'Used states are below the thresholds | '
 
-def check_states(lbpools, states_dict, default_state_limit, nagios_service,
-                 separator, warn, crit):
-    """ Compare the current states of each lbpool and compute results """
+        status += f'limit={state_limit}, current={cur_states}'
 
-    msg = ''
+        msg += ('{}\t{}\t{}\t{}{}').format(lbpool_name, nagios_service, exit_code, status, separator)
```
[the remark this answers is not in the document]
Comment (TuxPowered42) [reply src/lbpool_states.py:167]: Fixed.

**Pull request merged**